openedx_tagging/core/tagging/rules.py

```diff
@@ -124,5 +134,6 @@ def can_change_object_tag(
 rules.add_perm("oel_tagging.view_objecttag", rules.always_allow)
 
 # Users can tag objects using tags from any taxonomy that they have permission to view
+rules.add_perm("oel_tagging.view_objecttag_objectid", can_view_object_tag_objectid)
 rules.add_perm("oel_tagging.change_objecttag_taxonomy", can_view_taxonomy)
 rules.add_perm("oel_tagging.change_objecttag_objectid", can_change_object_tag_objectid)
```

Comment:
@rpenido What do you say to making the `oel_tagging.view_objecttag` rule mirror how `oel_tagging.change_objecttag` works?

So instead of explicitly checking the `view_objecttag_objectid` permissions in the retrieve() view, we call `view_objecttag` and let it check the (optional) taxonomy and `object_id` permissions.

I gave it a try here, and all your view tests still work: [diff](https://github.com/open-craft/openedx-learning/compare/rpenido/fal-3518-permissions-for-taxonomies...open-craft:openedx-learning:jill/rpenido/fal-3518-permissions-for-taxonomies)

Reply:
Good idea @pomegranited! I think this is good for consistency.
I picked your commit as it is and everything is fine: d54fb3a046d02a94e719fe681592e392c7bbfef7

Thank you for your work here! :smile: